src/components/TrafimageMaps/TrafimageMaps.scss

```diff
@@ -71,6 +71,9 @@
 
   .rs-popup {
     .rs-popup-container {
+      /* Overwrite with a very small minium width to allow small custom widths */
+      min-width: 100px;
+
```

Comment:
Core change to allow small popup widths. More changes see commit. All topic popups (inlcuding private topics) have been tested.